### ISSUE TITLE
Should these files be included in the repo?

### ISSUE THREAD

**Issue by ArtskydJ** [TehShrike/simplest-abstract-state-router-usage#1]
In [index.js lines 106 - 107](https://github.com/TehShrike/simplest-abstract-state-router-usage/blob/gh-pages/index.html#L106-L107) some files are grabbed from wzrd.in. It seems a little funny to have ractive.min.js hosted locally, but AST and ractive-state-router to be hosted elsewhere. If they were hosted locally too, then this demo could be used 100% offline, after someone cloned it.

I just loaded [the demo](http://tehshrike.github.io/simplest-abstract-state-router-usage/#/app/page1) and it took about a minute since ASR and/or RSR must not have been in the wzrd.in cache. (Now the page loads quickly as the files are in the cache again, but I think they clear the cache periodically.)

**Comment by TehShrike** [comment]
Yeah, they probably should be - the thinking was it's nice to be able to copy/paste the index.html file without having to copy many other files over
